api/admin/routes.py

```diff
@@ -1127,7 +1287,37 @@ def suppressed():
 @returns_json_or_response_or_problem_detail
 @requires_admin
 def genres():
-    """Returns a JSON representation of complete genre tree."""
+    """Returns a JSON representation of complete genre tree.
+    ---
+    get:
+      tags: 
+        - works
```

Comment:
Should probably be moved to the `standard lists` section we created in for SIMPLY-4165. Moving it there now will cause a PR conflict I think so we'll just have to resolve that when it comes time to merge.

Reply:
Merged develop into this branch to bring all commits up to date.